apps/dashboard/src/@/components/blocks/charts/area-chart.tsx

```diff
@@ -15,8 +22,14 @@ import {
   EmptyChartState,
   LoadingChartState,
 } from "../../../../components/analytics/empty-chart-state";
+import { cn } from "../../../lib/utils";
 
 type ThirdwebAreaChartProps<TConfig extends ChartConfig> = {
+  header?: {
+    title: string;
+    description?: string;
+    titleClassName?: string;
```

Comment:
When are titles styled differently?

Reply:
Need the titles to be a bit smaller in some places